src/unix/linux_like/linux/gnu/b32/hppa/mod.rs

```diff
@@ -0,0 +1,805 @@
+pub const ELIBBAD: ::c_int = 171;
+pub const ELIBSCN: ::c_int = 172;
+pub const ELIBMAX: ::c_int = 173;
+pub const ELIBEXEC: ::c_int = 174;
+pub const EILSEQ: ::c_int = 47;
+pub const ERESTART: ::c_int = 175;
+pub const ESTRPIPE: ::c_int = 176;
+pub const EUSERS: ::c_int = 68;
+pub const ENOTSOCK: ::c_int = 216;
+pub const EDESTADDRREQ: ::c_int = 217;
+pub const EMSGSIZE: ::c_int = 218;
+pub const EPROTOTYPE: ::c_int = 219;
+pub const ENOPROTOOPT: ::c_int = 220;
+pub const EPROTONOSUPPORT: ::c_int = 221;
+pub const ESOCKTNOSUPPORT: ::c_int = 222;
+pub const EOPNOTSUPP: ::c_int = 223;
+pub const EPFNOSUPPORT: ::c_int = 224;
+pub const EAFNOSUPPORT: ::c_int = 225;
+pub const EADDRINUSE: ::c_int = 226;
+pub const EADDRNOTAVAIL: ::c_int = 227;
+pub const ENETDOWN: ::c_int = 228;
+pub const ENETUNREACH: ::c_int = 229;
+pub const ENETRESET: ::c_int = 230;
+pub const ECONNABORTED: ::c_int = 231;
+pub const ECONNRESET: ::c_int = 232;
+pub const ENOBUFS: ::c_int = 233;
+pub const EISCONN: ::c_int = 234;
+pub const ENOTCONN: ::c_int = 235;
+pub const ESHUTDOWN: ::c_int = 236;
+pub const ETOOMANYREFS: ::c_int = 237;
+pub const ETIMEDOUT: ::c_int = 238;
+pub const ECONNREFUSED: ::c_int = 239;
+pub const EHOSTDOWN: ::c_int = 241;
+pub const EHOSTUNREACH: ::c_int = 242;
+pub const EALREADY: ::c_int = 244;
+pub const EINPROGRESS: ::c_int = 245;
+pub const ESTALE: ::c_int = 70;
+pub const EDQUOT: ::c_int = 69;
+pub const ENOMEDIUM: ::c_int = 182;
+pub const EMEDIUMTYPE: ::c_int = 183;
+pub const ECANCELED: ::c_int = 253;
+pub const ENOKEY: ::c_int = 184;
+pub const EKEYEXPIRED: ::c_int = 185;
+pub const EKEYREVOKED: ::c_int = 186;
+pub const EKEYREJECTED: ::c_int = 187;
+pub const EOWNERDEAD: ::c_int = 254;
+pub const ENOTRECOVERABLE: ::c_int = 255;
+pub const EHWPOISON: ::c_int = 257;
+pub const ERFKILL: ::c_int = 256;
+
+pub const SA_SIGINFO: ::c_int = 0x00000010;
+pub const SA_NOCLDWAIT: ::c_int = 0x00000080;
+
+pub const SOCK_STREAM: ::c_int = 1;
+pub const SOCK_DGRAM: ::c_int = 2;
+
+pub const F_GETLK: ::c_int = 5;
+pub const F_GETOWN: ::c_int = 11;
+pub const F_SETOWN: ::c_int = 12;
+
+pub const PTRACE_GETFPXREGS: ::c_uint = 18;
+pub const PTRACE_SETFPXREGS: ::c_uint = 19;
+pub const PTRACE_SYSEMU_SINGLESTEP: ::c_uint = 9;
+
+pub const MCL_CURRENT: ::c_int = 0x0001;
+pub const MCL_FUTURE: ::c_int = 0x0002;
+
+pub const POLLWRNORM: ::c_short = 0x100;
+pub const POLLWRBAND: ::c_short = 0x200;
+
+pub const EFD_NONBLOCK: ::c_int = 00200000;
+pub const SFD_NONBLOCK: ::c_int = 00200000;
+
+pub const SIGCHLD: ::c_int = 18;
+pub const SIGBUS: ::c_int = 10;
+pub const SIGUSR1: ::c_int = 16;
+pub const SIGUSR2: ::c_int = 17;
+pub const SIGCONT: ::c_int = 26;
+pub const SIGSTOP: ::c_int = 24;
+pub const SIGTSTP: ::c_int = 25;
+pub const SIGURG: ::c_int = 29;
+pub const SIGIO: ::c_int = 22;
+pub const SIGSYS: ::c_int = 31;
+pub const SIGSTKFLT: ::c_int = 7;
+#[deprecated(since = "0.2.55", note = "Use SIGSYS instead")]
+pub const SIGUNUSED: ::c_int = 31;
```

Comment:
Could you remove this? I think we don't need to add a deprecated one initially.

Reply:
What is the best way I can do this without loosing this pull request ?  Delete those lines in a new additional commit and push to the branch again?